src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -171,8 +174,8 @@ public boolean hasStarted() {
 
   public boolean isVisible() {
     return isDesktop() ?
-           usingImaBridge(webDriver -> webDriver.findElement(By.tagName("video")).isDisplayed()) :
-           driver.findElement(getVideoSelector()).isDisplayed();
+        usingImaBridge(webDriver -> webDriver.findElement(By.tagName("video")).isDisplayed()) :
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "video" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)